webhook_to_fedora_messaging_messages/github/__init__.py

```diff
@@ -0,0 +1,132 @@
+
```

Comment:
I would prefer if those functions were in a `utils.py` module rather than in the `__init__.py` to avoid possible circular dependencies.